tutorials/readme.md

```diff
@@ -0,0 +1,32 @@
+# WhizzML Tutorials
+
+This folder contains some basic scripts that are mainly written to provide
+simple examples of utilities that can be done using WhizzML. The examples
+are provided to illustrate how to build simple functions that can be combined
+to perform usual chores.
+
+Besides those examples, the `docs` folder contains `101's` commented examples
```

Comment:
maybe the folder could be called simply '101'? or is that weird?

Reply:
a bit, right? Also, if we decide to add other documents we can use the same folder...